pulseapi/entries/views.py

```diff
@@ -431,13 +430,9 @@ def post(self, request, *args, **kwargs):
                     encdata = thumbnail['base64']
                     proxy = ContentFile(base64.b64decode(encdata), name=name)
                     request_data['thumbnail'] = proxy
-            except:
+            except thumbnail.DoesNotExist:
```

Comment:
why are we catching this exception specifically (which isn't a valid exception thrown here btw)? I don't think we care about what kind of error it is because they are all Bad request errors. I think instead, we should just log the exception and respond with a bad request error instead of `return/pass`

Reply:
Ive changed the except's to 
```
except Exception as err:
                logging.error("No such entry",exc_info=err)
```
After making the changes in these two except's. I ran flake out and saw some errors in other files. Should I go ahead and make those changes as well?

Reply:
what are the other errors?

Reply:
Majority of them are:
```c
E501 line too long (80 > 79 characters)
```
Is the way i logged the errors correct?

Reply:
if it is in code that you've changed, yes, modify it so that this error goes away.